build/nuget-release.yml

```diff
@@ -0,0 +1,163 @@
+name: $(Package.Version)
+
+trigger: none
+
+variables:
+  - group: 'Arcus Event Grid - Integration Testing'
```

Comment:
Let's add a comment here that `Package.Version` is added on the build itself

Reply:
👍